src/index.html

```diff
@@ -3,11 +3,52 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, initial-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star "></div>
```

Comment:
Remove unnecessary spaces
```suggestion
      <div class="stars__star"></div>
```

Reply:
Corrections added in last comment "Removed unnecessary spaces ..."